ow_plexil/src/plans/TestPanTiltMoveCartesian.plp

```diff
@@ -16,5 +16,5 @@ TestPanTiltMoveCartesian:
   LibraryCall PanTiltMoveCartesian (Frame=BASE_LINK, X=1.46, Y=-0.4, Z=-0.05);
 
   // Point at Jupiter
```

Comment:
These comments would better be replaced by ROS messages saying what the antenna is pointing to.

Reply:
I don't think we should change test plan behavior in this PR. Please see my most recent thread comment.